lib/dor/services/client.rb

```diff
@@ -3,27 +3,13 @@
 require 'active_support/core_ext/hash/indifferent_access'
 require 'active_support/core_ext/module/delegation'
 require 'active_support/core_ext/object/blank'
-require 'deprecation'
 require 'cocina/models'
 require 'faraday'
 require 'singleton'
 require 'zeitwerk'
 
-class DorServicesClientInflector < Zeitwerk::Inflector
-  def camelize(basename, _abspath)
-    case basename
-    when 'sdr'
-      'SDR'
-    when 'version'
-      'VERSION'
-    else
-      super
-    end
-  end
-end
-
 loader = Zeitwerk::Loader.new
-loader.inflector = DorServicesClientInflector.new
+loader.inflector = Zeitwerk::GemInflector.new(__FILE__)
```

Comment:
@jcoyne Ooh, awesome, did dor-services-client pick up a version of Zeitwerk where this works as it should?

Reply:
It would have worked before too, if we didn't have to inflect `SDR`

Reply:
> And I would have gotten away with it too, if it weren't for you meddling kids

Reply:
in a perfect world, this would be a separate commit, but I don't care that much.

not sure with the case of version matters, but I wonder if it has to do with gem release?